### ISSUE TITLE
[Website] Google - No confirmation e-mail is received

### ISSUE THREAD

**Issue by sbadau** [mozilla/fx-private-relay#443]
**Affected versions**
Firefox 77.0.1
Firefox Private Relay 1.3.1

**Affected platforms**
Ubuntu 18.04

**Steps to reproduce**:
1. Launch Firefox.
2. Install the Firefox Private Relay add-on
3. Create a new relay alias
4. Navigate to Google.com, reach the sign-up form
5. Create an account using the alias from step 3.
6. Reach your mail address and look for the confirmation e-mail.

**Expected behavior**
A confirmation email should be forwarded to your mail address.

**Actual behavior**
No confirmation e-mail is received.

**Comment by groovecoder** [comment]
I reproduced this at first, but then I checked my spam folder and the confirmation email was in there. Did you check your spam folder?

**Comment by sbadau** [comment]
Yes, I did check the Spam folder (reproduced it on several Mozilla accounts). I have only seen this issue starting Wednesday. Until then I also received the confirmation emails in the Spam folder.
